package.json

```diff
@@ -71,7 +71,7 @@
         "cljs:lint": "clojure -M:clj-kondo --parallel --lint src --cache false",
         "ios:dev": "cross-env PLATFORM=ios gulp cap",
         "android:dev": "cross-env PLATFORM=android gulp cap",
-        "tldraw:build": "cd tldraw && yarn",
+        "tldraw:build": "cd tldraw && yarn install --network-timeout 100000",
```

Comment:
**non-blocking**

Better avoiding changing dev commands, I prefer adding the following to `~/.yarnrc`.

```
network-timeout 240000
```

Reply:
testing this solution instead. WIP until another passing build

Reply:
Thanks for the tip! that worked perfectly, and it's a much more elegant solution. 
Passing build: https://github.com/logseq/logseq/actions/runs/3951864705/jobs/6767902853

Reply:
Great work!